swarm-derive/Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "libp2p-swarm-derive"
 edition = "2021"
 rust-version = { workspace = true }
 description = "Procedural macros of libp2p-swarm"
-version = "0.34.0"
```

Comment:
`0.34.0` is already an unreleased version, you don't need to bump it :)

Reply:
I fixed the version bump. I'm not entirely sure what is the correct solution for `protocols/rendezvous/src/client.rs` The #into_out_event isn't in scope where the #generate_event_match_arm is. I'm not familiar with this code so I'm not very confident in making this change without more help from you.

Reply:
I'll fix it up tomorroe :)

Reply:
Please propose the change here so that I can learn about this code. I'll review and land it here.

Reply:
I pushed up two commits, hope you don't mind :)

Happy to explain in more details on what happens there!